src/analyze_model.py

```diff
@@ -0,0 +1,195 @@
+import numpy as np
+
+
+def build_in_out_region_map(num_r_in, num_c_in, num_r_out, num_c_out):
+    '''
+    Description:
+        Given an input image (num_r_in, num_c_in) and an bounding box
+        output (num_r_out, num_c_out), build a map that can convert
+        inputs row/cols to bound box rows/cols
+    Args:
+        num_r_in (int): number of rows in input
+        num_c_in (int): number of columns in input
+        num_r_out (int): number of rows in bounding box output
+        num_c_out (int): number of columns in bounding box output
+    Returns:
+        region_map_r (np.array shape=[num_r_in,]): maps initial input rows
+            to the binned rows of the output r_out = region_map_r[r_in]
+        region_map_c (np.array shape=[num_c_in,]): maps initial input columns
+            to the binned rows of the output c_out = region_map_c[c_in]
+    Updates:
+        N/A
+    Write to file:
+        N/A
+    '''
+    # divide up input into regions
+    delta_r = np.floor(float(num_r_in) / float(num_r_out))
```

Comment:
You take ints and convert to floats immediately. Is that just for "correct" division? If so there are better solutions.